app/main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        super().learn_skill("JavaScript")
+        super().learn_skill("HTML")
+        super().learn_skill("CSS")
```

Comment:
Consider using the `extend()` method or `+=` operator to add multiple skills to the `skills` list at once, as per the checklist requirement.